src/util.cpp

```diff
@@ -415,6 +415,11 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
                             ", <i><a href=\"https://www.qt.io\">https://www.qt.io</a></i></p>"
                             "<p>Opus Interactive Audio Codec"
                             ", <i><a href=\"https://www.opus-codec.org\">https://www.opus-codec.org</a></i></p>"
+#    if defined( _WIN32 ) && !defined( WITH_JACK )
```

Comment:
Hmm. Although not related, I think we should hava a "WITH_ASIO" default compile parameter instead of checking WITH_JACK everywhere...

Reply:
Perhaps introduce the "WITH_ASIO" as part of audio redesign? Current reasoning is that the ASIO is just the default and therefore no need to define. Thinking if there will be a "no sound" option, it might help to add a "WITH_ASIO" define.